client/src/components/UserForm.css

```diff
@@ -9,6 +9,10 @@
   margin-bottom: 25px;
 }
 
+.container-input {
+  width: 50%;
+}
+
 .container-btn {
   text-align: center;
   margin-bottom: 50px;
```

Comment:
Bootsrap class `mb-[0-5]`. Better to use one of their predefiened values to ensure consistency throughout the app.
https://getbootstrap.com/docs/4.0/utilities/spacing/

Reply:
I'm using that now.